src/MessageList.jsx

```diff
@@ -0,0 +1,36 @@
+import Card from "@mui/material/Card";
```

Comment:
Great work splitting the form form the list shows that you understand where the data is coming from and where its actually required.